src/pages/api/auth-connect/verify-validation-code.ts

```diff
@@ -2,9 +2,11 @@ import {
   addIdentityToUser,
```

Comment:
uuuh I don't get your point

Reply:
Well, instead of having 4 `.catch()` that do the same thing, i.e.
```ts
foo().catch(
  if (error instanceof GraphqlErrors) {
    throw webErrorFactory(webErrors.identityNotFound);
  }

  if (error instanceof ConnectUnreachableError) {
    throw webErrorFactory(webErrors.connectUnreachable);
  }

  throw error;
);

bar().catch(
  if (error instanceof GraphqlErrors) {
    throw webErrorFactory(webErrors.identityNotFound);
  }

  if (error instanceof ConnectUnreachableError) {
    throw webErrorFactory(webErrors.connectUnreachable);
  }

  throw error;
);

jean().catch(
  if (error instanceof GraphqlErrors) {
    throw webErrorFactory(webErrors.identityNotFound);
  }

  if (error instanceof ConnectUnreachableError) {
    throw webErrorFactory(webErrors.connectUnreachable);
  }

  throw error;
);

michel().catch(
  if (error instanceof GraphqlErrors) {
    throw webErrorFactory(webErrors.identityNotFound);
  }

  if (error instanceof ConnectUnreachableError) {
    throw webErrorFactory(webErrors.connectUnreachable);
  }

  throw error;
);
```

we would have

```ts
try {
  foo();
  bar();
  jean();
  michel();
} catch (error) {
 if (error instanceof GraphqlErrors) {
    throw webErrorFactory(webErrors.identityNotFound);
  }

  if (error instanceof ConnectUnreachableError) {
    throw webErrorFactory(webErrors.connectUnreachable);
  }

  throw error;
}
```

Reply:
ah, yes, didn't see it in my review 😅 
I agree

Reply:
This one will be done on another PR, we need to improve our DB command errors before changing to try/catch.